src/H5FDpublic.h

```diff
@@ -138,6 +138,78 @@
  */
 #define H5FD_FEAT_DEFAULT_VFD_COMPATIBLE 0x00008000
 
+/* ctl function definitions: */
+#define H5FD_CTL_OPC_RESERVED 512 /* Opcodes below this value are reserved for library use */
+#define H5FD_CTL_OPC_EXPER_MIN                                                                               \
+    H5FD_CTL_OPC_RESERVED /* Minimum opcode value available for experimental use                             \
+                           */
+#define H5FD_CTL_OPC_EXPER_MAX                                                                               \
+    (H5FD_CTL_OPC_RESERVED + 511) /* Maximum opcode value available for experimental use */
+
+/* ctl function op codes: */
+#define H5FD_CTL__INVALID_OPCODE              0
+#define H5FD_CTL__TEST_OPCODE                 1
+#define H5FD_CTL__GET_MPI_COMMUNICATOR_OPCODE 2
+#define H5FD_CTL__GET_MPI_RANK_OPCODE         3
+#define H5FD_CTL__GET_MPI_SIZE_OPCODE         4
+#define H5FD_CTL__NUM_OPCODES                 5 /* must be last */
```

Comment:
Should decide if we want to keep this.  If memory serves, this #define  is not used at present.  I believe I was thinking of using it to validate op-codes, but I'm not sure the idea makes any sense at present.

Reply:
I don't think it's necessary unless we are going to iterate over all possible opcodes.

Reply:
Removed this definition for now. If we need it later we can add it back, but for now there's no need to add it to our public definitions if it's not being used anywhere.